src/cpp/server/server.cc

```diff
@@ -287,12 +287,14 @@ void Server::Wait() {
 }
 
 void Server::PerformOpsOnCall(CallOpBuffer* buf, Call* call) {
-  static const size_t MAX_OPS = 8;
-  size_t nops = MAX_OPS;
-  grpc_op ops[MAX_OPS];
-  buf->FillOps(ops, &nops);
-  GPR_ASSERT(GRPC_CALL_OK ==
-             grpc_call_start_batch(call->call(), ops, nops, buf));
+  if (call->call()) {
```

Comment:
We are getting here in the case reported in issue #914. The thread is shutting down, so I don't think that there's a stuckness issue. Perhaps there is a better solution to the problem.